config/initializers/cors.rb

```diff
@@ -7,7 +7,7 @@
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins "localhost:3001"
+    origins "subvisuwheel.netlify.app"
```

Comment:
This will make it so that only the frontend app at that URL is able to call the backend app. Probably not what we want as we also need to keep `localhost:3001` for local development

Look into how to allow multiple origins for this

Reply:
Okay, I will :+1: